src/data_types/guid.rs

```diff
@@ -102,7 +31,7 @@ impl fmt::Display for Guid {
 /// ```
 pub unsafe trait Identify {
     /// Unique protocol identifier.
-    const GUID: Guid;
+    const UNIQUE_GUID: Guid;
```

Comment:
Yeah I got it after having changed it, my bad because I thought I already made the change. Btw I renamed the constant field because of the clash occuring with the `GUID` type generated by the `uefi-sys` crate. Perhaps renaming it to `IDENTITY_GUID` would be good enough, what do you think?

Reply:
Ah, I didn't realize constant names were in the same namespace as type names.

Is it in any way possible to change the name of the `uefi-sys` type? Like `Guid` or something?
I'd rather not change this protocol's associated constant name.